components/director/internal/repo/list.go

```diff
@@ -5,39 +5,29 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/kyma-incubator/compass/components/director/internal/persistence"
-	"github.com/kyma-incubator/compass/components/director/pkg/pagination"
 	"github.com/pkg/errors"
+
+	"github.com/kyma-incubator/compass/components/director/internal/persistence"
 )
 
-type PageableQuerier struct {
+type Lister struct {
```

Comment:
now it looks that you modified PageableQuerier, but you just renamed a file. Can you check if some git magic tricks exist to correctly show your changes?

Reply:
I divided this PR into a few commits for easier reviewing